specs/MultiProfile.md

```diff
@@ -118,6 +124,46 @@ HRESULT AppWindow::OnCreateCoreWebView2ControllerCompleted(HRESULT result, ICore
     // ...
 }
 ```
+
+### Access and use cookie manager from profile
+
+```cpp
+wil::com_ptr<ICoreWebView2CookieManager> m_cookieManager;
+ScenarioCookieManagement::ScenarioCookieManagement(ICoreWebView2Controller* controller)
+{
+    wil::com_ptr<ICoreWebView2> coreWebView2;
+    CHECK_FAILURE(controller->get_CoreWebView2(&coreWebView2));
+    auto webview7 = coreWebView2.try_query<ICoreWebView2_7>();
+    if (webview7)
+    {
+        wil::com_ptr<ICoreWebView2Profile> profile;
+        CHECK_FAILURE(webview7->get_Profile(&profile));
+        auto profile2 = profile.try_query<ICoreWebView2Profile2>;
+        if (profile2)
+        {
+            CHECK_FAILURE(profile2->get_CookieManager(&m_cookieManager));
```

Comment:
I don't understand why we have to go `WebView2` -> `Profile` -> `CookieManager`. The text said that you could get there directly via `ICoreWebView2.get_CookieManager`.

It seems the purpose of this new API is to provide a more roundabout way of doing something you can already do in a simpler way.

Reply:
The first reason is as mentioned in the previous conversation, we will support cookie manager without creating WebView2s in the long term, so we provide this interface in profile, then the users can get it without creating a WebView2.
Another reason is, from a design perspective, the cookie manager should be bound with a profile, not a WebView2.

Reply:
Okay, this is moving the cookie manager so that the Profile is the primary source, and the existing WebView2 is now just a convenience function that gets the cookie manager from the profile. May want to call this out in the prologue: "Providing the CookieManager from the profile is more logical, and it sets the groundwork for allowing an app to manage the cookies of a profile without having to create a WebView2 first."

Reply:
Thanks for your suggestion! Will add it to the prologue later.

Reply:
Updated the prologue. Please let me know if there are any problems.